modules/log6x-configuring-otlp-output.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.0/log6x-clf.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log6x-configuring-otlp-output_{context}"]
+= Configuring OTLP output
+
+The OpenTelemetry Protocol (OTLP) output allows the cluster administrator to collect and forward logs to OTLP receivers. The OTLP output uses the protocol defined by the https://opentelemetry.io/docs/specs/otlp/[OpenTelemetry Observability Framework] to send data over HTTP with JSON encoding.
+
+:FeatureName: The OTLP output log forwarder
+include::snippets/technology-preview.adoc[]
+
+.Procedure
+
+* To enable OTLP, create or edit the `ClusterLogForwarder` CR using the template below:
++
+.Example ClusterLogForwarder CR
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  annotations:
+    observability.openshift.io/tech-preview-otlp-output: "enabled" # <1>
+  name: clf-otlp
+spec:
+  serviceAccount:
+    name: <service_account_name> 
+  outputs:
+  - name: otlp
+    type: otlp
+    otlp:
+      tuning:
+        compression: gzip 
+        delivery: atLeastOnce
+        maxRetryDuration: 20
+        maxWrite: 10M
+        minRetryDuration: 5
+      url: <otlp-url> # <2>
+  pipelines:
+  - inputRefs:
+    - application
+    - infrastructure
+    - audit
+    name: otlp-logs
+    outputRefs:
+    - otlp
+----
+<1> This annotation enables the OpenTelemetry Protocol (OTLP) output, which is a technical preview feature.
+<2> This URL must be absolute and is a placeholder for the OTLP endpoint where logs are sent.
+
+[NOTE]
+====
+The OTLP output uses the OpenTelemetry data model, which is different from other log output formats. The OTLP output does not send ViaQ logs. Instead, it adheres to the https://opentelemetry.io/docs/specs/semconv/[OpenTelemetry Semantic Conventions] and https://opentelemetry.io/docs/specs/otel/logs/data-model/[Logs Data Model].
```

Comment:
The OTLP output uses the OpenTelemetry data model, which is different from the ViaQ data model used by other   output types.  It adheres to the OpenTelemetry Protocol using sementtic conventions defined by the OpenTelemetry framework.